boolean/parser/9d298219-6b80-41a1-aae7-2f8907a76c32.json

```diff
@@ -4,13 +4,15 @@
     "inputs": {
       "ed9fa3f5-96fa-4e06-94b3-fde00cca3166": {
         "type": "any",
```

Comment:
I think this should be "string". Also "input value" as a description is not very meaningful - something like "String to be parsed" would be better

Reply:
we need to modify the implementation in that case, as for now, it works like this: `outputs.boolean(!!data, tag);` - the input string`"false"` would evaluate to `true`

Reply:
Ah right. Ok - then let's rename it to "boolean/Evaluator" and have the port description say eg. "Data to be evaulated as boolean"